modules/cloud-credential-operator.adoc

```diff
@@ -32,5 +314,5 @@ No configuration required.
 [discrete]
 == Notes
 
-* The Cloud Credential Operator uses credentials from `kube-system/aws-creds`.
-* The Cloud Credential Operator creates secrets based on `credentialsrequest`.
+* The CCO uses credentials from `kube-system/aws-creds`.
```

Comment:
this line is true for AWS, but the secrets are different for each platform we support (eg kube-system/azure-credentials for Azure).

Reply:
Good point. This notes section actually seems perhaps a bit less necessary now that the CCO operation is described in more detail. What do you think about just deleting it?

Reply:
seems reasonable to remove it

Reply:
seems reasonable to remove it.